### ISSUE TITLE
make source-manifests is failing with 'make: *** [Makefile:79: source-manifests] Error 2'

### ISSUE THREAD

**Issue by andrewschoen** [openshift/ocs-operator#174]
This has been failing for a couple days now in our downstream CI. 

Relevant log files, using commit 2c0a70a038473e3e492bbcc3e73d5a883c51987a:

```
+ export ROOK_IMAGE=quay.io/rhceph-dev/rook:4.2-153.37a9e6b0.master
+ ROOK_IMAGE=quay.io/rhceph-dev/rook:4.2-153.37a9e6b0.master
++ ./get-latest.py rhceph-dev/mcg-operator
+ export NOOBAA_IMAGE=quay.io/rhceph-dev/mcg-operator:4.2-87
+ NOOBAA_IMAGE=quay.io/rhceph-dev/mcg-operator:4.2-87
+ make source-manifests
Sourcing CSV and CRD manifests from component-level operators
hack/source-manifests.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   627    0   627    0     0   3215      0 --:--:-- --:--:-- --:--:--  3198

 29 68.5M   29 20.3M    0     0  17.7M      0  0:00:03  0:00:01  0:00:02 17.7M
 42 68.5M   42 29.2M    0     0  13.6M      0  0:00:05  0:00:02  0:00:03 9048k
 48 68.5M   48 33.1M    0     0  10.5M      0  0:00:06  0:00:03  0:00:03 6551k
 53 68.5M   53 36.7M    0     0  9074k      0  0:00:07  0:00:04  0:00:03 5594k
 59 68.5M   59 41.0M    0     0  8156k      0  0:00:08  0:00:05  0:00:03 5284k
 68 68.5M   68 47.2M    0     0  7861k      0  0:00:08  0:00:06  0:00:02 5495k
 74 68.5M   74 51.3M    0     0  7356k      0  0:00:09  0:00:07  0:00:02 4534k
 81 68.5M   81 56.1M    0     0  7048k      0  0:00:09  0:00:08  0:00:01 4695k
 87 68.5M   87 59.6M    0     0  6675k      0  0:00:10  0:00:09  0:00:01 4683k
 90 68.5M   90 61.8M    0     0  6238k      0  0:00:11  0:00:10  0:00:01 4263k
 96 68.5M   96 66.0M    0     0  6067k      0  0:00:11  0:00:11 --:--:-- 3858k
100 68.5M  100 68.5M    0     0  6068k      0  0:00:11  0:00:11 --:--:-- 3983k
make: *** [Makefile:79: source-manifests] Error 2
```

**Comment by andrewschoen** [comment]
Actually, the first failure like this was seen yesterday at commit a4a888a7bed9c46b17d6249b556118723b401cd5

**Comment by jarrpa** [comment]
@kshlm @davidvossel

**Comment by davidvossel** [comment]
The noobaa/mcg operator image is failing to dump the csv. This is a noobaa regression. 

```docker run --rm -it quay.io/rhceph-dev/mcg-operator:4.2-87 olm csv yaml
PANI[0000] ☠️  Panic Attack: [] invalid configuration: no configuration has been provided 
panic: (*logrus.Entry) (0x182df40,0xc00019b420)

goroutine 1 [running]:
github.com/sirupsen/logrus.Entry.log(0xc0000b6420, 0xc00042fc50, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/vendor/github.com/sirupsen/logrus/entry.go:239 +0x2cf
github.com/sirupsen/logrus.(*Entry).Log(0xc00019acb0, 0xc000000000, 0xc0000dfd90, 0x1, 0x1)
	/vendor/github.com/sirupsen/logrus/entry.go:268 +0xc8
github.com/sirupsen/logrus.(*Entry).Logf(0xc00019acb0, 0x0, 0x187a9c7, 0x1d, 0xc0000dfe40, 0x2, 0x2)
	/vendor/github.com/sirupsen/logrus/entry.go:314 +0xfc
github.com/sirupsen/logrus.(*Entry).Panicf(0xc00019acb0, 0x187a9c7, 0x1d, 0xc0000dfe40, 0x2, 0x2)
	/vendor/github.com/sirupsen/logrus/entry.go:352 +0x65
github.com/noobaa/noobaa-operator/pkg/util.Panic(0x1ab0180, 0xc0003f7800)
	/pkg/util/util.go:409 +0x105
github.com/noobaa/noobaa-operator/pkg/util.CurrentNamespace(0x0, 0x0)
	/pkg/util/util.go:450 +0x111
github.com/noobaa/noobaa-operator/pkg/options.init.0()
	/pkg/options/options.go:90 +0x2c4
```

**Comment by andrewschoen** [comment]
Thanks @davidvossel. Have you opened an issue with the noobaa devs about this? If not, could you point me at the right place to do that? Thanks!

**Comment by davidvossel** [comment]
> Have you opened an issue with the noobaa devs about this? If not, could you point me at the right place to do that? Thanks!

hey, i haven't filed an issue. This is the repo the issue is occurring in. https://github.com/noobaa/noobaa-operator Filing an issue there would make sense.

**Comment by guymguym** [comment]
Hi guys. This is indeed a new issue which will fix soon.
In the meanwhile see the suggested workaround here: https://github.com/noobaa/noobaa-operator/issues/67#issuecomment-536910325

**Comment by andrewschoen** [comment]
@guymguym @davidvossel Thanks for help! I've got the workaround in place in our CI and it's working. Once we've got an official fix on https://github.com/noobaa/noobaa-operator/issues/67 I'll remove that.

**Comment by b-ranto** [comment]
@guymguym @davidvossel I was wondering where we could expect the fix for this issue? i.e. will it land in ocs or noobaa operator? It could help us prepare for the fix, i.e. to make sure the fix won't break our CI again.

**Comment by guymguym** [comment]
@b-ranto a proper fix will be made in noobaa-operator.
You can track it in the upcoming release v2.0.1 
See https://github.com/noobaa/noobaa-operator/milestone/1

**Comment by guymguym** [comment]
BTW I just noticed there's an even easier workaround which is just to set the `WATCH_NAMESPACE` env to anything:
```
docker run --rm -it -e WATCH_NAMESPACE=xxx quay.io/rhceph-dev/mcg-operator:4.2-87 olm csv
```

**Comment by b-ranto** [comment]
FYI: This can be closed, after rebase to noobaa 2.0.1, we are no longer hitting this issue.

**Comment by jarrpa** [comment]
Resolved by #179.
